### PR TITLE
feat: do not load fonts by default

### PR DESCRIPTION
**Pull request by rolznz**
Fixes https://github.com/getAlby/bitcoin-connect/issues/31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -439,6 +439,19 @@ html {
 
 > 💡 using near-white or black brand colors? either set a lower `bc-brand-mix` or make sure to use an off-white for `bc-color-brand` and off-black for `bc-color-brand-dark` to avoid conflicts with the modal background color.
 
+### Fonts
+
+By default Bitcoin Connect does not use custom fonts for reduced footprint and privacy, however custom fonts can be used by providing CSS for the `Inter` (sans serif) and `Roboto Mono` (monospace) families. For example:
+
+```js
+for (const family of ['Inter', 'Roboto Mono']) {
```
Comment (rolznz) [inline README.md:447]: Maybe this should be plain html links, not JS

Comment (reneaaron) [reply README.md]: So you would just provide the default Google Font embed code? 

```
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&family=Roboto+Mono:ital,wght@0,100..700;1,100..700&display=swap" rel="stylesheet">
```

Comment (rolznz) [reply README.md]: added, thanks!

**Review by reneaaron — APPROVED** [review]
Looks good to me! 

utACK

**Comment by bumi** [comment]
<3

**Pull request merged**